brvs-rules/src/main/java/iroha/validation/rules/impl/BlockRule.java

```diff
@@ -0,0 +1,104 @@
+package iroha.validation.rules.impl;
+
+import iroha.protocol.Commands;
+import iroha.protocol.Commands.Command;
+import iroha.protocol.QryResponses.QueryResponse;
+import iroha.protocol.TransactionOuterClass.Transaction;
+import iroha.validation.rules.Rule;
+import java.io.StringReader;
+import java.security.KeyPair;
+import javax.json.Json;
+import javax.json.JsonReader;
+import jp.co.soramitsu.iroha.java.IrohaAPI;
+import jp.co.soramitsu.iroha.java.Query;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+public class BlockRule implements Rule {
```

Comment:
As I understood this is a rule validating the minimal asset balance for the account. Am I correct? If so it should be renamed

Reply:
Please, add comment describing the logic